src/index.html

```diff
@@ -6,12 +6,173 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+            name="feedback"
```

Comment:
The 'name' attribute should be unique for each input field. Please change the 'name' attribute for the Surname input field to something like 'surname'.

Reply:
<img width="928" alt="image" src="https://user-images.githubusercontent.com/70220667/234487319-0f9b29a2-7e17-49b5-8658-86e530dc2dbc.png">
fix it  everywhere